modules/ROOT/pages/configure-service-mesh.adoc

```diff
@@ -35,108 +36,144 @@ After the new adapter is ready, the old pod is terminated and all requests are r
+Before	you can start managing your API using Service Mesh, you must prepare your namespace by provisioning a Service Mesh Adapter. Provisioning prepares the namespace for the Service Mesh configuration. After you provision the adapter, you must redeploy your existing application to ensure that the Envoy sidecar is injected with each POD.
 
 To provision the Anypoint Service Mesh adapter:
  
 . Determine the adapter size based on the plan.
-For more information, see...
 . From the Service Mesh installer download location in the Command window, type the following command to create the adapter:
 +
 [source,text,linenums]
 ----
-./service-mesh adapter create \--name=anypoint-service-mesh-adapter \--namespace=marketing \
-
---size=medium \--clientId=<replace_with_clientId> \
-
---clientSecret=<replace_with_clientSecret> \
-
---platformUri=https://anypoint.mulesoft.com
+./service-mesh adapter create 
+--name=<adapter name>
+--namespace=<adapter namespace>
+--size=<adapter plan size>
+--clientId=<clientId of the application> 
+--clientSecret=<clientSecret of the application> 
+--platformUri=<URL of Anypoint Platform>
 ----
-+	
-Where:
 +
 [%header%autowidth.spread]
 |===
 | Parameter Name | Description | Required/Optional | Default Value
-| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. For the Beta exercise, use “anypoint-service-mesh-adapter” for this parameter. | Required | N/A
-| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. For the Beta exercise, use “marketing” for this parameter | Optional | default
+| name | The name of the adapter that you want to create, for example, “service-mesh-test-adapter”. | Required | N/A
+| namespace | The namespace in which you want to create the adapter, for example, “mulesoft-apps”. | Optional | default
 | size | The size of the adapter based on the number of CPUs and memory to use | Optional | Medium
 | clientId | The client ID of the application | Required | N/A
 | clientSecret | The client secret of the application | Required | N/A
 | platformUri | The URL of the host where Anypoint Platform is installed | Optional | https://anypoint.mulesoft.com
 |===
 +
 . Verify the adapter status:
++   
 `./service-mesh adapter list`
 +
 image::provision-service-mesh-adapter-output.png[Example Output]
 +
-. Review adapter logs to see detailed information about Kubernetes events and adapter transactions.
+. Review adapter logs to see detailed information about Kubernetes events and adapter transactions:
++
 `./service-mesh adapter logs --name=<adapter_name> --namespace=<namespace>`
 
+== Create a non-Mule API
+
+If your MuleSoft or non-MuleSoft application does not exist, you must create it before you bind the service to the adapter. 
```

Comment:
Please remove this statement or reword "If your MuleSoft or non-MuleSoft application does not exist"
I would say, now we will create an API in API Manager and then connect that API to a service